.gitignore

```diff
@@ -79,3 +79,8 @@ vendor
 notes.txt
 image_move.sh
 .DS_Store
+template-processors/applier/Dockerfile
```

Comment:
Why are we ignoring these Dockerfiles?

Reply:
Those are generated now from Dockerfile.in files in build process.